python/pyarrow/src/arrow/python/arrow_to_pandas.cc

```diff
@@ -2523,7 +2523,8 @@ Status ConvertCategoricals(const PandasOptions& options, ChunkedArrayVector* arr
   }
   if (options.strings_to_categorical) {
     for (int i = 0; i < static_cast<int>(arrays->size()); i++) {
-      if (is_base_binary_like((*arrays)[i]->type()->id())) {
+      if (is_base_binary_like((*arrays)[i]->type()->id()) ||
```

Comment:
The binary_view changes are tangential to pandas 3.x right? I wonder if they shouldn't be done in their own PR

Reply:
Somewhat tangential yes, as in that it is also a useful change to do regardless of the other changes here. I am also not entirely sure there is a very specific test for this. 

Happy to move out to a separate PR, although I then would like both to get merged for 19.0

Reply:
Yea I would prefer to separate out so we can analyze test coverage too. I suppose that would also be helpful if these changes end up going through to different releases (although that's not the aim)

Reply:
OK. it turns out to not actually affect the tests here, so good to move that to a separate PR with actual test coverage: https://github.com/apache/arrow/pull/45176

Reply:
I've merged the related issue and marked as 19.0.0